sample-apps/fdb-profile-analyzer/Dockerfile

```diff
@@ -1,20 +1,29 @@
 FROM python:3.9-slim
 USER root
 
-RUN microdnf --enablerepo=ubi* --disablerepo=rhel* install -y wget \
-                         bind-utils \
+RUN apt-get update \
+    && apt-get  install -y wget \
+                         bind9-utils \
                          lsof  \
+                         less \
                          net-tools \
+                         jq \
                          openssl \
-                         pkg-config \
-                         procps-ng\
-                         iputils && \
-     microdnf --enablerepo=ubi* --disablerepo=rhel* clean all \
+                         pkg-config
 
+COPY prepare_fdb_lib.sh /tmp/
 RUN groupadd -g 4059 foundationdb
-RUN useradd -m -d / -s /bin/bash -u 4059 -o foundationdb -g 4059 -G 266
+RUN useradd -m -d / -s /bin/bash -u 4059 -o foundationdb -g 4059
+RUN mkdir /app/
+RUN chown foundationdb:foundationdb -R /app/
 RUN pip install foundationdb
 RUN pip install dateparser
-RUN rpm -ivh https://github.com/apple/foundationdb/releases/download/7.1.19/foundationdb-clients-7.1.19-1.el7.x86_64.rpm
+RUN chmod 755 /tmp/prepare_fdb_lib.sh
+RUN /tmp/prepare_fdb_lib.sh 6.3.23 && \
```

Comment:
I believe we don't need those additional libraries and we should inject the required libraries during runtime with an init container, the same way we do it with the operator and all other tooling.

Reply:
Agree. Adding the init container.